client/components/gravatar-with-hovercards/styles.scss

```diff
@@ -0,0 +1,87 @@
+.gravatar-hovercard {
+	// Ensure our styles take precedence over the default styles
+	.gravatar-hovercard__inner {
+		.gravatar-hovercard__primary-blog-card {
+			background: var( --color-main-background );
+			border: 1px solid var( --color-border-subtle );
+			/* stylelint-disable-next-line scales/radii */
+			border-radius: 6px;
+			padding: 12px;
+			box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+			margin-top: 12px;
+
+			.gravatar-hovercard__primary-blog-card-header {
+				display: flex;
+				align-items: center;
+				margin-bottom: 12px;
+			}
+
+			.gravatar-hovercard__primary-blog-card-site-icon.reader-avatar {
+				margin-right: 12px;
+				display: inline-block;
+				max-height: unset;
+
+				.site-icon {
+					border-radius: 4px;
+				}
+			}
+
+			.gravatar-hovercard__primary-blog-card-site-info {
+				flex-grow: 1;
+				min-width: 0; // Enable text truncation
+			}
+
+			.gravatar-hovercard__primary-blog-card-site-title {
+				font-size: 14px;
+				font-weight: 600;
+				line-height: 1.3;
+				margin: 0;
+				color: var( --color-text );
+				overflow: hidden;
+				text-overflow: ellipsis;
+				white-space: nowrap;
+			}
+
+			.gravatar-hovercard__primary-blog-card-username {
+				font-size: 12px;
+				line-height: 1.4;
+				margin: 2px 0 0;
+				color: var( --color-text-subtle );
+				overflow: hidden;
+				text-overflow: ellipsis;
+				white-space: nowrap;
+			}
+
+			.gravatar-hovercard__primary-blog-card-description {
+				font-size: 13px;
+				line-height: 1.4;
+				margin: 0 0 16px;
+				color: var( --color-text );
+				overflow: hidden;
+				text-overflow: ellipsis;
```

Comment:
The ellipsis and punctuation look a little odd with RTL, but could be a follow-up:

<img width="357" alt="Screenshot 2025-06-18 at 1 38 25 PM" src="https://github.com/user-attachments/assets/dee20d25-400c-454d-8b90-8e301c8340de" />

Reply:
Im not sure I follow. If it is actually a RTL language then shouldn't those be on the left as shown? I could be missing something 😅

Reply:
You're right, but we don't do that for the post content:

<img width="830" alt="Screenshot 2025-06-18 at 2 01 04 PM" src="https://github.com/user-attachments/assets/9f9af103-ce07-49f5-b65a-7f50c401ca5f" />

I'm not sure if that's a feature or a bug. 😅 It does make sense for untranslated English to stay LTR.

Reply:
It looks like we have the same issue in the reader profile page header too 🤔 
![Screenshot 2025-06-19 at 7 14 26 AM](https://github.com/user-attachments/assets/8b6f9fbf-844d-408d-994f-bb3b8b46b01e)

Reply:
I think I see what the issue is and a way to resolve. I will explore in a follow up to clean that up for the card, site/feed stream description header, and reader profile description.